Reading Legacy PK1 from PKXDelta / PikaSav

ReignOfComputer opened the pull request.

ReignOfComputer committed:
Reading Legacy PK1 from PKXDelta/PikaSav

Part 1 before I go to work. Todo TID and bits stuff.

ReignOfComputer committed:
Reading Legacy PK1 from PKXDelta / PikaSav

Completed.

ReignOfComputer committed:
Oops, handle PikaSav Party Pokemon

kwsch commented:
Wouldn't it be better to create a method that rearranges the data into the Rhydon pk1 format rather than having separate methods to handle loading? That would keep the conversion code separate from the loading code, plus you wouldn't duplicate the loading code.

ReignOfComputer committed:
Rearranging legacy structures to PK1

The more optimized route. No idea what 0x00, 0x02, and 0x24-0x2E are for
though.

ReignOfComputer commented:
Wasn't thinking, there we go :)
